fastapi/models/fastapi_endpoint.py

```diff
@@ -80,10 +80,11 @@ def _check_root_path(self):
 
     @api.depends("root_path")
     def _compute_urls(self):
+        base_url = self.env["ir.config_parameter"].sudo().get_param("web.base.url")
         for rec in self:
-            rec.docs_url = f"{rec.root_path}/docs"
-            rec.redoc_url = f"{rec.root_path}/redoc"
-            rec.openapi_url = f"{rec.root_path}/openapi.json"
+            rec.docs_url = f"{base_url}{rec.root_path}/docs"
+            rec.redoc_url = f"{base_url}{rec.root_path}/redoc"
+            rec.openapi_url = f"{base_url}{rec.root_path}/openapi.json"
```

Comment:
Vetry strange since I tested the UI on V14 without these changes and everything is working.

Reply:
Except that I added the base_url by hand ;-)